### ISSUE TITLE
CIFAR10 example does not work properly to me.

### ISSUE THREAD

**Issue by watermelon8855** [LaurentMazare/tch-rs#244]
I was trying cifar-10 example and the accuracy got stuck between 7.40% ~ 7.60% even after > 100 epochs.
I tried to resolve the problem by reinstalling torchlib, tch-rs, cuda, etc.. but none of them worked.
I even tried training on different device, but the problem was still there.

What could be the reason??

Environment - 1 
> CUDA 10.2
> Windows 10
> GTX 1070
> tch = "0.2.0"

Environment - 2
> CUDA 10.2
> Windows 10
> GTX 970
> tch = "0.2.0"

**Comment by LaurentMazare** [comment]
Thanks for reporting this issue. I think this is a regression that was introduced when switching to 1.6 and broke the C++ function setting the learning rate. I just pushed some fix that should get this back to working. Please update to the latest github version, give it another try and let us know if this still fails to train (you should see > 50% after one epoch).

**Comment by watermelon8855** [comment]
```
epoch:    1 test acc: 60.67%
epoch:    2 test acc: 71.65%
epoch:    3 test acc: 72.22%
epoch:    4 test acc: 72.32%
epoch:    5 test acc: 71.14%
epoch:    6 test acc: 10.00%
epoch:    7 test acc: 10.00%
epoch:    8 test acc: 10.00%
```
```
epoch:    1 test acc: 50.75%
epoch:    2 test acc: 70.29%
epoch:    3 test acc: 75.72%
epoch:    4 test acc: 10.00%
```
```
epoch:    1 test acc: 10.00%
epoch:    2 test acc: 10.00%
```

These are results what I've got. I've tested multiple times, and sometimes it works for a short time, but it quickly fails again.

**Comment by LaurentMazare** [comment]
That's weird, I'm afraid I won't be very helpful here as I've tried on two different boxes and both went through the 150 epochs and reached the expected accuracy. Both boxes are using libtorch 1.6.0 pre-built from the pytorch.org website (one is cuda 9.2 and the other cuda 10.2). I would suggest doing a cargo clean, ensuring that you have the proper libtorch version but I imagine you already went through this.

**Comment by watermelon8855** [comment]
I failed to resolve the issue on my GTX 970 machine, but GTX1070 machine seems to peform well now. It is indeed strange.. Thanks for your support!

**Comment by LaurentMazare** [comment]
Closing for now as there hasn't been any development over the last weeks. Feel free to re-open if this causes some issues again.
